lib/test.js

```diff
@@ -471,6 +471,54 @@ Test.prototype.notDeepLooseEqual
     });
 };
 
+Test.prototype.lt
+= Test.prototype.lessThan
+= function (a, b, msg, extra) {
+    this._assert(a < b, {
+        message : defined(msg, 'should be less than'),
```

Comment:
it seems like this default message should maybe be more like `inspect(a) + ' should be less than ' + inspect(b)`?

Reply:
I thought of that, but the reason why I ended up not implementing it was because none of the other functions in-line the `b` argument like this. But if you feel it makes sense here, I'll happily update it 😃